src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/PaperNotificationService.java

```diff
@@ -68,12 +69,12 @@ private boolean shouldPrintNotificationForRespondentSolicitor(CaseDetails caseDe
     public void printApplicantRejectionGeneralApplication(CaseDetails caseDetails, String authToken) {
         CaseDocument applicantGeneralApplicationRejectDoc = rejectGeneralApplicationDocumentService.generateGeneralApplicationRejectionLetter(
             caseDetails, authToken, APPLICANT);
-        bulkPrintService.sendDocumentForPrint(applicantGeneralApplicationRejectDoc, caseDetails);
+        bulkPrintService.sendDocumentForPrint(applicantGeneralApplicationRejectDoc, caseDetails, CCDConfigConstant.APPLICANT);
```

Comment:
same here static import

Reply:
can't do- as one already with same in static import